crates/cdk/src/nuts/nut00.rs

```diff
@@ -46,18 +57,26 @@ pub enum Error {
     NUT11(#[from] crate::nuts::nut11::Error),
 }
 
-/// Blinded Message [NUT-00]
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+/// Blinded Message (also called `output`)
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct BlindedMessage {
     /// Amount
+    ///
+    /// The value for the requested [BlindSignature]
     pub amount: Amount,
-    /// Keyset Id
+    /// Keyset ID
+    ///
+    /// ID from which we expect a signature.
     #[serde(rename = "id")]
     pub keyset_id: Id,
     /// Blinded secret message (B_)
+    ///
+    /// The blinded secret message generated by the sender.
     #[serde(rename = "B_")]
-    pub b: PublicKey,
+    pub secret: PublicKey,
```

Comment:
I'm not sure that we should rename this to secret as its actually the blinded secret and not the secret itself. Though a more descriptive name maybe a good idea `blinded_secret`?